changing-data-model.rst

```diff
@@ -0,0 +1,90 @@
+How to add changes to the Central Subject Registry (CSR) data model?
+====================================================================
+
+This page describes the changes that are required to modify the CSR data model.
+
+
+Adding new entity
+*****************
+
+Changes in `csr` package:
+-------------------------
+
+The data model is defined as a data class in `<csr/csr.py>`_.
+In order to add a new entity, a new class has to be defined.
+
+Classes are defined using `pydantic` - library to validate JSON documents and convert them into Python objects, 
+based on Python Type Annotations.
+
+All new entities have to have a non-nullable `identity` defined. They also have to be a part of `SubjectEntity` union 
+and `CentralSubjectRegistry` class defined in `csr.py`.
+If there is a relation between the new entity and other entities, it has to be specified as a value of `references` key 
+in Schema dictionary.
+
+For more details see `official pydantic documentation`_.
+
+.. _`official pydantic documentation`: https://pydantic-docs.helpmanual.io/
+
+New entities has to be added to `<csr/subject_registry_writer.py>`_ and `<csr/subject_registry_reader.py>`_.
+
+Changes in `csr2transmart` package:
+-----------------------------------
+
+In order to include observations for the new entity when mapping input data to the TranSMART data model, 
```

Comment:
This part is not required anymore, right?

Reply:
oh no, I lost one commit during rebasing... I will remove this part.